### DISCUSSION TITLE
Seeing this ERROR: unsupported Unicode escape sequence when submitting repo

### DISCUSSION THREAD

**Discussion by turner-strongdm** [artifacthub/hub#1995] [Q&A] [answered]
Currently trying to submit a repo to artifact hub. I'm seeing `error registering package sdm-relay version 0.1.0: ERROR: unsupported Unicode escape sequence (SQLSTATE 22P05)` whenever it process the repo. I'm not sure where the error is occurring. It works if I add the repo manually to helm, but was trying to list on Artifacthub.

Repo Link: [https://github.com/strongdm/charts]
Link supplied to Artifact Hub: [https://helm.strongdm.com/stable]
Github link for the `index.yaml`: https://github.com/strongdm/charts/blob/main/stable/index.yaml

**Comment by turner-strongdm** [comment, marked as answer]
I found my own answer. It appears the files were getting encoded in `UTF-16`. As soon as I updated the encoding for all the files to `UTF-8` it worked. I have my vscode set to `UTF-8`, but it appears that me using remote WSL2 instance for my vscode may be the culprit. Will do more research.

**Reply by tegioz** [reply]
Awesome, thanks for sharing the answer! 👍
